src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -22,54 +21,28 @@
   >
     <template #pre-title="{ closeNav, isOpen, currentBreakpoint, className }" v-if="displaySidenav">
       <div :class="className">
-        <transition name="sidenav-toggle">
-          <div
-            v-show="sidenavHiddenOnLarge"
```

Comment:
I only deleted the `sidenavHiddenOnLarge` data property for the localnav, since it's not needed anymore. But we still need it in the navigator to be able to keep open or close the navigator.